ws_types.go

```diff
@@ -109,9 +109,9 @@ type (
 	}
 
 	WsOrder struct {
-		Order           WsBasicOrder `json:"order"`
-		Status          string       `json:"status"`
-		StatusTimestamp int64        `json:"statusTimestamp"`
+		Order           WsBasicOrder     `json:"order"`
+		Status          OrderStatusValue `json:"status"`
```

Comment:
What do we achieve with this change if we do not add the enumerated values (i.e `OrderStatusFilled`, `OrderStatusPending`...) ?

Another couple notes:

- The type, I would name it `OrderStatus` The "value" suffix sounds meaning. "Enum" could be better if `OrderStatus` were already taken.
- While I understand the general inertia of creating types for this situation, unless you add methods to them (i.e `IsPending`, IsFilled`, `Is(other OrderStatus) bool`), it's preferrable to have a couple of constants as strings pinpointing the enumerated values, as passing around this types normally require type coercion when the actual string value is needed.

Reply:
I agree it would be better if we have enums but they are not there. I simply re-use the type that was used for https://github.com/sonirico/go-hyperliquid/blob/e8dcd99718f19009abe4f599f9ee5b4f88a947a5/types.go#L277-L291

Which allows my editor to correctly auto-complete predefined values.